### ISSUE TITLE
FEAT: minimum width for tabs

### ISSUE THREAD

**Issue by mcanepa** [beekeeper-studio/beekeeper-studio#1401]
If I open too many tabs, I reach a point where I no longer know what they are about (see screenshot)

They shoud have a minimum width so they can display it's name

![Captura de pantalla de 2022-10-18 12-34-04](https://user-images.githubusercontent.com/25961439/196495841-796ff3ba-8128-464e-b308-0785fc30b6b2.png)

**Comment by rathboma** [comment]
Yes, this is something we have to deal with. We previously had an overflow area, but then it wasn't obvious how many tabs were open, so this was the compromise for the time being.

We're probably not going to substantially change this anytime soon, but if someone creates a PR with a solution they think is better I'd be happy to consider it.

**Comment by mcanepa** [comment]
Thanks for your feedback

> but then it wasn't obvious how many tabs were open

I think this should be done with #1403 in mind, so the whole thing looks something like this:

![65159595-07c93d80-da3d-11e9-9e47-159c9fc3693f](https://user-images.githubusercontent.com/25961439/196511866-59333022-1889-4f19-b597-fa45c9886429.gif)

**Comment by vespadj** [comment]
`tabs-header > nav`, tabs should be in more than one line.
This can be rapid tested by edit 
`<ul class="nav-tabs nav">`
to
`<ul class="nav">`

at

`#page-content > div > div.tabs-header > ul`
XPath: `/html/body/div/div[1]/div[2]/div[1]/div[4]/div/div[1]/ul`

some cosmetic fixes of the CSS class will be required.

<img width="1914" height="488" alt="Image" src="https://github.com/user-attachments/assets/6cadc624-7d2a-417b-acdc-817a3cb88594" />

**Comment by mcanepa** [comment]
the multiline idea is horrible UX
